src/domain/notifications/index.types.d.ts

```diff
@@ -26,7 +26,6 @@ type LnInvoicePaidUsdWalletArgs = {
   paymentHash: PaymentHash
   recipientWalletId: WalletId
   cents: UsdCents | bigint
-  displayCurrencyPerSat?: DisplayCurrencyPerSat
```

Comment:
Why remove? How does this work if display currency is CRC?

Reply:
because in this case we dont need display currency, in the next PR we will use an unified API for both (BTC and USD) and we will have again display currency for usd as base